README.md

```diff
@@ -558,6 +558,8 @@ const uiSchema = {
 };
 ```
 
+In addition it is valid to mark an array as "required".  This means the user must select at least one item to add to the array.
```

Comment:
Indeed that's a serious diff. Not sure I am comfortable with that but I guess we can't have the silver bullet.